forge-gui/res/cardsfolder/upcoming/gwenna_eyes_of_gaea.txt

```diff
@@ -0,0 +1,10 @@
+Name:Gwenna, Eyes of Gaea
+ManaCost:2 G
+Types:Legendary Creature Elf Druid Scout
+PT:2/3
+A:AB$ Mana | Cost$ T | Produced$ Combo Any | Amount$ 2 | RestrictValid$ Spell.Creature,Activated.Creature | SpellDescription$ Add two mana in any combination of colors. Spend this mana only to cast creature spells or activate abilities of a creature or creature card.
+T:Mode$ SpellCast | ValidCard$ Creature.powerGE6 | ValidActivatingPlayer$ You | TriggerZones$ Command | Execute$ TrigCounter | TriggerDescription$ Whenever you cast an artifact spell with mana value 6 or greater, draw a card.
```

Comment:
Wrong TriggerDescription?

Reply:
Yeah I've fixed Gwenna and carrion card a few minutes ago